integration_tests/tests/temporary-accommodation/assess/assess.cy.ts

```diff
@@ -100,7 +100,7 @@ context('Apply', () => {
             // Given there are assessments in the database
             const assessments = assessmentSummaryFactory.buildList(6, { status: factoryStatus })
             const searchedForReferral = assessments[1]
-            const searchCRN = searchedForReferral.person.crn
+            const searchQuery = searchedForReferral.person.query
```

Comment:
Will definitely need for the API to be updated and offer new types for this to be merged, but this will have to be done with a deprecation step -- i.e. API should accept both `crn` and `query` parameters, for this to be merged, then the `crn` parameter should be retired from the API, to ensure no loss of functionality without having to synchronise deploys (which isn't very doable, unfortunately!)